cmake/modules/FindARIA.cmake

```diff
@@ -0,0 +1,70 @@
+# Filename: FindARIA.cmake
+#
+# Usage:
+#   find_package(ARIA [REQUIRED] [QUIET])
+#
+# Once done this will define:
+#   ARIA_FOUND       - system has ARIA MAgicCrypt0
+#   ARIA_INCLUDE_DIR - the include directory containing ARIA
+#   ARIA_LIBRARY     - the path to the libARIA library
+#
+
+set(ARIA_INCLUDE_DIR)
+set(ARIA_LIB_FILE)
+
+# when debugging cmake, ARIA_DIR environment variable can be manually set here:
+# set(ENV{ARIA_DIR} "/mnt/c/workspace/MagicCrypto")
+# set(ENV{ARIA_DIR} "c:\\workspace\\MagicCrypto")
+
+# Make sure we have a ARIA_DIR encironment variable with the path to MagicCrypto
+if ("$ENV{ARIA_DIR}" STREQUAL "")
+    message(ERROR "ERROR: FindARIA.cmake missing ARIA_DIR value")
+    message(STATUS "Please set ARIA_DIR environment variable path to your MagicCrypto")
+else()
+    set(ARIA_INCLUDE_DIR "$ENV{ARIA_DIR}/include")
+    message(STATUS "FindARIA.cmake found ARIA_INCLUDE_DIR = $ENV{ARIA_DIR}")
+    # set(ARIA_LIBRARY "$ENV{ARIA_INCLUDE_DIR}/lib")
+endif()
+
+# Check that the appropriate files exist
+find_path(ARIA_INCLUDE_DIR NAMES "mcapi.h" )
+
+if (EXISTS "${ARIA_INCLUDE_DIR}/mcapi.h")
```

Comment:
Negate these three checks to simplify and remove the commented out block.

Reply:
logic is now "if not exists.... error"